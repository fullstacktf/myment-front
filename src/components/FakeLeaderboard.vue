<template>
    <div class="containerRanking">
        <div class="titleWrapper"><h3>RANKING</h3></div>
        <div class="containerLeader">
            <div class="section">Pos 1: </div>
            <div class="section">user 2</div>
            <div class="section">user 3</div>
            <div class="section">user 4</div>
            <div class="section">user 5</div>
            <div class="section">user 6</div>
            <div class="section">user 7</div>
            <div class="section">user 8</div>
            <div class="section">user 9</div>
            <div class="section">user 10</div>
        </div>
    </div>
</template>

<script>
export default {

}
</script>

<style scoped>
@import url('https://fonts.googleapis.com/css?family=Merriweather+Sans&display=swap');

.containerRanking{
    height: 90%;
    
}
.containerLeader{
    display: grid;
    grid-template-rows: repeat(10, 10%);
    overflow: hidden;
    margin-top: 15px;
    width: 90%;
    height: 90%;
    padding: 10px;
}
.section{
    display: grid;
    align-items: center;
    justify-content: center;
    margin: 4px;
    background: white;
    border-radius: 20px;
    opacity: 0.5;
    box-shadow: 3px 3px 10px black;

}
h3{
    text-align: center;
    font-family: 'Merriweather Sans', sans-serif;
;
}
.titleWrapper{
    width: 90%;
}
/*
@keyframes marquee {
    0%   { transform: translate(0, 0); }
    100% { transform: translate(0, -620%); }
}
*/
</style>
