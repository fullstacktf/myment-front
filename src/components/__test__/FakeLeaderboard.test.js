import FakeLeaderboard from '../FakeLeaderboard.vue';
import { shallowMount, createLocalVue } from '@vue/test-utils';
import Buefy from 'buefy';
import mock from '../../test/mockBack';

describe('FakeLeaderboard', () => {
  let localVue;
  beforeEach(() => {
    const Vue = createLocalVue();
    Vue.use(Buefy);
    localVue = Vue;
  });
  test('render component', () => {
    const wrapper = shallowMount(FakeLeaderboard, {
      localVue,
    });
    expect(wrapper.html()).toMatchSnapshot();
  });

  test('Check props', () => {
    
  });

  test('check Contents props', () => {
    
  });
});


