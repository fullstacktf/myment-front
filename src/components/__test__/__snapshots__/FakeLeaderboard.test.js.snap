// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FakeLeaderboard render component 1`] = `"<div class=\\"containerRanking\\"><div class=\\"titleWrapper\\"><h3>RANKING</h3></div> <div class=\\"containerLeader\\"><div class=\\"section\\">Pos 1: </div> <div class=\\"section\\">user 2</div> <div class=\\"section\\">user 3</div> <div class=\\"section\\">user 4</div> <div class=\\"section\\">user 5</div> <div class=\\"section\\">user 6</div> <div class=\\"section\\">user 7</div> <div class=\\"section\\">user 8</div> <div class=\\"section\\">user 9</div> <div class=\\"section\\">user 10</div></div></div>"`;
